constitution.tex

```diff
@@ -520,9 +520,10 @@
 \asubsubsection{The Introductory Process}
 The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
 \asubsubsubsection{The Evaluation Period}
-The Process will begin between the first and fourth week of each semester, then last between six (6) and ten (10) weeks, starting when the Evaluations Director initiates it.
+The Process will begin between the first and second week of each semester, then last six (6) weeks, starting when the Evaluations Director initiates it.
 The Executive Board may hold a closed ballot Simple Majority vote to extend the Introductory Process or the Introductory Packet.
 If an Introductory Process is extended, the Introductory Evaluation shall occur at the termination of the extended Process, which may be outside of the period defined in \ref{Introductory Evaluation}.
+The Introductory Process may be held twice a semester at the discretion of the evaluations director. If the evaluations director deems it necessary, the second intro process must begin within a week of the first process ending, and must last the same six (6) weeks.
```

Comment:
`within a week of the first process ending` when is this? How does this factor in time for appeals?

Each sentence gets its own line.
```suggestion
The Introductory Process may be held twice a semester at the discretion of the evaluations director.
If the evaluations director deems it necessary, the second intro process must begin within a week of the first process ending, and must last the same six (6) weeks.
```

Maybe this could be wrapped into the earlier portion of this paragraph, it seems a little bit out of place, and the `same six (6) weeks` is weird.